src/styles/blocks/calendar.scss

```diff
@@ -0,0 +1,58 @@
+@import "../utils/variables";
+
+.calendar {
+  display: flex;
+  justify-content: center;
+
+  &__month {
+    display: flex;
+    flex-wrap: wrap;
+    gap: $gap-between-days;
+    padding: 10px;
+    width: calc(#{$day-size} * 7 + #{$gap-between-days} * 14);
```

Comment:
![image](https://user-images.githubusercontent.com/55781950/226853501-7af13dad-24b7-41e7-ab47-f0f03929ca43.png)
When set to monday, width only includes 6 days in a week

Reply:
On second look, it only includes 6 days in a week with all settings